### PR TITLE
Rest: use auth parameters from action inputs

### PR DESCRIPTION
**Pull request by 0lvin**

### CODE REVIEW

File: cloudify_rest/tasks.py [hunk 1]
```diff
@@ -102,8 +103,11 @@ def _execute(params, template_file, instance, node, save_path=None,
         merged_params = {}
         merged_params.update(node.properties.get("params", {}))
         merged_params.update(params)
+        merged_auth = node.properties.copy()
+        if auth:
+            merged_auth.update(auth)
```
Comment (mabuaisha) [inline cloudify_rest/tasks.py]: @0lvin you are trying to update `merged_auth` which is a dict with `boolean` value `auth` , I think this should be updated like this

```
if auth:
    merged_auth.update({YOUR_KEY: auth})
```

File: cloudify_rest/tasks.py [hunk 2]
```diff
@@ -93,7 +94,7 @@ def execute_as_relationship(params=None, template_file=None, save_path=None,
 
 
 def _execute(params, template_file, instance, node, save_path=None,
-             prerender=False, remove_calls=False):
+             prerender=False, remove_calls=False, auth=False):
```
Comment (mabuaisha) [inline cloudify_rest/tasks.py]: @0lvin Do you mean here to pass `auth=None` instead of `auth=False` ?

File: cloudify_rest/tasks.py (hunk 1 above)
Comment (0lvin) [reply cloudify_rest/tasks.py]: Boolean - is only placeholder for something that is not true/filled by default.

File: cloudify_rest/tasks.py (hunk 2 above)
Comment (0lvin) [reply cloudify_rest/tasks.py]: Updated in 309bf0a

**Pull request merged**